lib/logstash/filters/translate.rb

```diff
@@ -62,11 +65,20 @@ class LogStash::Filters::Translate < LogStash::Filters::Base
   #     merci: gracias
   #     old version: new version
   #     
-  # NOTE: it is an error to specify both `dictionary` and `dictionary_path`
+  # NOTE: it is an error to specify both `dictionary` and `dictionary_path` or `dictionary_url`
   config :dictionary_path, :validate => :path
 
-  # When using a dictionary file, this setting will indicate how frequently
-  # (in seconds) logstash will check the YAML file for updates.
+  # The full URI path of a Web service who generates an yml format response. 
+  # The response generated needs to be equals than needed for @dictionary_path
+  #
+  # NOTE: it is an error to specify both `dictionary` and `dictionary_path` or `dictionary_url`
+  config :dictionary_url, :validate => :string
+
+  # Filename (without extension) where .yml will be stored from a Web service.
+  config :file_to_download, :validate => :string, :default => "dictionary"
```

Comment:
I was thinking more about it.
If I want to maintain the same behaviour than load_file I musn't use random file.
I will use base64 hash on the path, and the result will be the filename. 
If I do the change with this behaviour I don't need to be care about delete files moreover when the logstash could be restarted, or on refresh.

What do you think? I will try to do the change right now.